index.go

```diff
@@ -881,12 +904,7 @@ func (r *indexReader) Series(ref uint64, lbls *labels.Labels, chks *[]ChunkMeta)
 			MaxTime: maxt,
 		})
 	}
-
-	if valid, err := r.checkCRC(d1.be32(), int(ref)+sl, l); !valid {
```

Comment:
Are we checking it anywhere in Series? Or we just don't want to do it, because it is already checked somewhere in  `decbuf`?

Reply:
The checksums are now validated as part of `decbufAt`/`decbufUvarintAt`